vj4/locale/zh_CN.yaml

```diff
@@ -774,3 +774,20 @@ home_domain_account: 当前域的设置
 'The value `{1}` of {0} already exists.': '{0} 的值 `{1}` 已经存在。'
 Display name {1} you want to set is used by others.: 您想要设置的显示名 {1} 已经被其他人使用了。
 Numeric PID: 数字题号
+Only {0} problems can be copied in one request, we got {1}.: 一次请求只能复制 {0} 个题目，但是你输入了 {1} 个。
+Problem is successfully copied.: 题目复制完成。
+problem_copy: 复制题目
+Copy Problem: 复制题目
+Be Copied: 被复制
+Copy from: 复制自
+Problem ID List: 题目 ID 列表
+Copy: 复制
+Destination Domain: 目标域
+Domain ID: 域 ID
+Test data comes from: 测试数据来自
+From other problem: 来自其他题目
+Copy Problem To: 将题目复制到
+What's this?: 这是什么？
+About test data: 关于测试数据
+With this feature, you can copy problems that you can view from a domain to a specific domain. Title, content and other properties will be copied. Of course, test data is not included.: 你可以通过这个功能将某域下你能查看的题目复制到指定的域中，题目的标题、描述等将被复制过来。当然，测试数据不包括在其中。
+Although test data will not be copied, test data of copied problem will be linked to the source problem if source has test data uploaded. By uploading new test data, it will not be linked anymore. Permission of linked test data follows the original one.: 虽然测试数据不会被复制，但是在原题有数据的情况下，系统会在复制后将题目的测试数据链接到原题。在题目设置页面中可以通过上传新的测试数据的方式，来取消这个数据链接。链接后的测试数据的权限以原题为准。
```

Comment:
Instead of copying the test data directly, the test data of the copied problems will be linked to the test data of the source problems (called the source test data). Thus, the copied problems can observe the changes in the source test data. The permissions of the test data of the copied problems follow the source test data, e.g., you still might not download them but the judges can. By uploading some new test data, the link will be broken and the new test data will be used.